tests/forms_tests/field_tests/test_filefield.py

```diff
@@ -80,3 +80,32 @@ def test_disabled_has_changed(self):
 
     def test_file_picklable(self):
         self.assertIsInstance(pickle.loads(pickle.dumps(FileField())), FileField)
+
+    def test_multiple(self):
+        f = FileField(multiple=True)
+        files = [SimpleUploadedFile('file_%s' % i, ('%s' % i).encode('utf-8')) for i in range(2)]
+        self.assertEqual(files, f.clean(files))
+
+        # ensure validation is executed for all files
+        f = FileField(multiple=True)
+        files = [SimpleUploadedFile('file_%s' % i, b'') for i in range(2)]
+        with self.assertRaisesMessage(ValidationError, 'The submitted file is empty.'):
+            f.clean(files)
+
+        def test_validator(value):
+            if not isinstance(value, SimpleUploadedFile):
+                raise ValidationError('Value is not a file.')
+
+        f = FileField(multiple=True, validators=[test_validator])
+        files = [SimpleUploadedFile('file_%s' % i, ('%s' % i).encode('utf-8')) for i in range(2)]
```

Comment:
Didn't you want to actually test that the validator raises `ValidationError` here?

Reply:
oh, yes good catch